arch/Kconfig

```diff
@@ -715,6 +715,20 @@ config FPU_SHARING
 	  across context switches to allow multiple threads to perform concurrent
 	  floating point operations.
 
+config FP_USE_SOFT_FLOAT
+	bool "Use Software Floating Point Operations"
+	depends on !FPU && TOOLCHAIN_SUPPORTS_SOFT_FLOAT
+	help
+	  Enable using software floating point operations supported by
+	  toolchain.
+
+config TOOLCHAIN_SUPPORTS_SOFT_FLOAT
```

Comment:
This doesn't belong in `arch/Kconfig`.

Should go into a toolchain Kconfig, like here:
https://github.com/zephyrproject-rtos/sdk-ng/blob/master/cmake/zephyr/Kconfig

To bad Zephyr SDK 0.12 has just been released.

Reply:
I have added the kconfig there.